cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -38,7 +38,8 @@ func (p *diskInspectionProcessor) process(pd persistentDisk,
 
 	ir, err := p.inspectDisk(pd.uri)
 	if err != nil {
-		return pd, err
+		// Don't directly return err to avoid terminating the import.
```

Comment:
Log the error?

Reply:
It has been print to console; to record in trace log, I need to allow appending trace log. done.